### ISSUE TITLE
Google Safe Browsing / Web Risk API Monitor

### ISSUE THREAD

**Issue by alexschomb** [louislam/uptime-kuma#3300]
### ⚠️ Please verify that this feature request has NOT been suggested before.

- [X] I checked and didn't find similar feature request

### 🏷️ Feature Request Type

New Monitor

### 🔖 Feature description

All major web browsers are integrating "Safe search" with the Google Safe Browsing API, and thus block access to websites if Google decides those to be harmful. There are some paid SaaS that provide monitors for checking the generally free API. The [Google Safe Browsing API](https://developers.google.com/safe-browsing) is free for non-commercial use, while [Google Web Risk API](https://cloud.google.com/web-risk)  lookups are free up to 100.000 checks per month. I think that supporting such API lookups could be a nice addition to the already existing HTTP/HTTPS monitor feature of uptime-kuma.

### ✔️ Solution

Implementation of a new Google Safe Browsing and/or Google Web Risk API monitor. In the backend it might be solved as an additional checkbox for the HTTP/HTTPS monitors and a general settings tab for the API credentials needed.

**Comment by CommanderStorm** [comment]
Out of curiosity: Why is this relevant to you? (has one of your websites been flagged as a false-positive/...)

**Comment by alexschomb** [comment]
Correct, I just had a website being flagged as false-positive (couldn't find any traces of malicious code – currently under review by Google). I would like to be informed of such issues instead of finding out about them randomly.
